website/content/docs/sync/github.mdx

```diff
@@ -21,6 +21,7 @@ Prerequisites:
   [access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
   which has access to the repository you want to manage secrets for and write permissions on the repository's actions secrets.
   In the list of GitHub permissions, this is simply called "Secrets". Choosing this will automatically include read-only Metadata.
+  Alternatively, you can set up a [GitHub application](#github-application).
```

Comment:
Should we slightly rephrase this so the application is suggested as the preferred method?

Should we add an info box explaining some of the pitfalls of the access tokens such as no automatic renewals and automatic revocation if the creator of the token is removed from the GH org?